queries.md

```diff
@@ -4,76 +4,96 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+filter: {name: 'Babelgum'}
+project: {name:1}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+filter: {number_of_employees: {\$gt: 5000} }
+project:
+sort: {number_of_employees:1}
+Collaboration:
+Limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+filter: {$and: [{founded_year:{$gte:2000}}, {founded_year:{\$lte:2005}}]}
+project: {name:1, founded_year:1}
 
 ### 4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
 
-<!-- Your Code Goes Here -->
+filter: {$and: [{"ipo.valuation_amount": {$gt:100000000}, founded_year: {\$lt:2010}}]}
+project: {name:1, ipo:1}
+
+Limit: 20
 
 ### 5. All the companies that have less than 1000 employees and have been founded before 2005. Order them by the number of employees and limit the search to 10 companies.
 
-<!-- Your Code Goes Here -->
+filter: {$and: [{number_of_employees:{$lt:1000}}, {founded_year:{\$lt:2005}}]}
+sort: {number_of_employees: 1}
+limit: 10
 
 ### 6. All the companies that don't include the `partners` field.
 
-<!-- Your Code Goes Here -->
+find: {partners: {\$size:0}}
```

Comment:
more correct here would be:
```js
 {partners: {$exists: false}}
```

Reply:
I see!! I found that, but thought the array bit was more JS-like